src/lib/viz/index.ts

```diff
@@ -23,3 +23,6 @@ export {
 
 // Basemap helpers
 export { createGoogleMap, createMap } from './basemap';
+
+// Widgets
+export { CategoryWidget } from './widgets';
```

Comment:
I think we can already start with a `carto.widget` namespace, for a Category widget class there, as you proposed... what do you think?

Reply:
I think we should move them when the namespace refactor happens :)